cores/src/category.rs

```diff
@@ -6,22 +6,19 @@ use crate::openai_utils::generate_embedding;
 #[derive(sqlx::FromRow, Debug)]
 pub struct Category {
     pub category_id: Uuid,
-    pub markdown_id: Uuid,
-    pub category: Option<String>,
+    pub category: String,
     pub embedding: Option<Vec<f64>>,
 }
 
 pub async fn store_in_category_table(
     pool: &PgPool,
-    markdown_id: Uuid,
     category: &str,
 ) -> color_eyre::Result<Category> {
     let embedding = generate_embedding(category).await?;
     let result = sqlx::query_as!(
         Category,
-        "INSERT INTO Category (category_id, markdown_id, category, embedding) VALUES ($1, $2, $3, $4) RETURNING *",
+        "INSERT INTO categories (category_id, category, embedding) VALUES ($1, $2, $3) RETURNING *",
```

Comment:
Do you think we don't need to have `markdown_id` anymore? The reason why we needed the `markdown_id` is because we wanted a `FK` reference to generate `Categories` to a certain `MD_id`

Reply:
We don't need _this_ FK cause we already have another one 🎉 

There is the `markdown_categories` table that has two columns `category_id` and `markdown_id`

And we use this to create the links! This way a Markdown can link to multiple categories in the DB! [We don't do this yet but I think we will soon!]

Reply:
Oh okay, so it's like a `JOIN` Table?